README.md

```diff
@@ -126,15 +126,44 @@ LOG_LEVEL=trace npm run test
 
 Alternatively, one may set `LOG_LEVEL` in the `.env` file.
 
-### API Keys
-
-As a temporary measure, API keys from a file are used for authentication. To generate five keys, for example:
-
-    touch secret_api_keys.txt
-    chmod 600 secret_api_keys.txt
-    for i in {1..5}; do LC_ALL=C tr -dc a-zA-Z0-9 </dev/random | LC_ALL=C head -c 80; echo ''; done > secret_api_keys.txt
-
-Set `API_KEYS_FILE` in `.env` to `secret_api_keys.txt` (or whatever was specified above) but remember also to set `API_KEYS_FILE=test_keys.txt` (verbatim) in .env.test such that the .env value will not take precedence over the non-existent test value. In other words, `.env` and `.env.test` get merged in tests, and if you set API_KEYS_FILE in `.env`, the tests will unintentially pick up the wrong keys if you do not also set the variable correctly in `.env.test`. By default, if neither `.env` nor `.env.test` specifies `API_KEYS_FILE` the tests will use `test_keys.txt` which are not intended for production use.
+### Authentication and Authorization
+
+A valid Bearer JSON Web Token (Bearer JWT) is required in requests to the PDC service. The exception is the API documentation page. These environment variables configure the expected realm and JWKS issuer keys:
+- `AUTH_SERVER_ISSUER`: the URL for the issuer (this is the "realm" URL in Keycloak terminology).
+- `AUTH_SERVER_JWKS_URI`: the URI for this service to get the JSON Web Key Sets (JWKS) to validate the signature of the Bearer JWT against the public key of the issuer. Set this to `${AUTH_SERVER_ISSUER}/protocol/openid-connect/certs` if Keycloak is your auth service.
```

Comment:
I've found that it is clearer and more sustainable to put environment variable documentation in the `.env.example` instead of the README. It's closer to what it's documenting, it's more likely to be updated correctly as environment variables change, and it simplifies the instructions in the README. I did that recently in the [data-viewer `.env.example`](https://github.com/PhilanthropyDataCommons/data-viewer/blob/main/.env.example), for reference.